chat-plugins/calculator.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+function isNumeric(str) {
+	return !isNaN(parseFloat(str)) && isFinite(str);
+}
+
+function clean(arr) {
+	let i;
+	for (i = 0; i < arr.length; i++) {
+		if (arr[i] === "") {
+			arr.splice(i, 1);
```

Comment:
LGTM says: Removing an array item without adjusting the loop index 'i' causes the subsequent array item to be skipped.

Reply:
Function is removed. See below.